docs/integrations-in-rancher/rancher-extensions.md

```diff
@@ -91,6 +101,13 @@ You must reload the page after disabling extensions or display issues may occur.
 
 :::
 
+## Enabling Unauthenticated Access to an Extension
+
+In Rancher v2.9.0, you can allow unauthenticated access to an extension. You may want to enable unauthenticated access if the extension enables a new locale or adds custom branding. By default, all extensions require user authentication to load. 
+
+To enable unauthenticated access to an extension, set `plugin.noAuth` to `true` in the CR used by the extension.
+
+
```

Comment:
@diogoasouza Remove this extra empty line

Reply:
Done